docs/administration.rst

```diff
@@ -1521,6 +1521,16 @@ The following parameters have to be set in the configuration files.
         If set to True, Entrust will use the primary client ID of 1, which applies to most use-case.
         Otherwise, Entrust will first lookup the clientId before ordering the certificate.
 
+.. data:: ENTRUST_DEFAULT_EKU
```

Comment:
shall we also document `DIGICERT_CIS_USE_CSR_FIELDS`?

Reply:
Agreed, since there isn't any documentation on CIS options currently, I documented them in a separate PR https://github.com/Netflix/lemur/pull/4685